apps/fluent-tester/src/TestComponents/Button/ButtonTest.tsx

```diff
@@ -53,7 +54,7 @@ const buttonSections: TestSection[] = [
     component: E2EButtonTest,
   },
   ...Platform.select({
-    android: [null], //Following sections are not supported from Fluent Android
+    android: [], //Following sections are not supported from Fluent Android
```

Comment:
Replaced [null] with [] because I got an error on iOS "Each child in a list should have a unique "key" prop" and this change got rid of the error